keepassxc-browser/keepassxc-browser.js

```diff
@@ -15,6 +15,9 @@ var _detectedFields = 0;
 // Element id's containing input fields detected by MutationObserver
 var _observerIds = [];
 
+// Document URL
+let _documentURL = document.location.href;

```

Comment:
Careful using `let`, it's not supported on some older browsers, and will result in a syntax error. https://caniuse.com/#feat=let

Reply:
I don't see an issue? There are 0.25% usage of desktop browsers that don't support it. And I believe of that percentage, 80% is IE which doesn't support WebExtensions.

Reply:
True. My main reservation was in trying to keep it consistent with the rest of the codebase, which seems to use `var`.

Reply:
I need to change this behaviour in the future. `var` is preferred when the variable must be used globally in the other scripts. `let` keeps the scope in one file.